Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.7.8
+COPY config /config
+COPY writer /writer
+COPY app.rb /
+COPY config.ru /
+COPY queue_io.rb /
+COPY Gemfile /
+COPY Gemfile.lock /
+RUN bundle install
+WORKDIR /
+ENV PORT 3000
```

Comment:
This is probably not needed? But all of our other Docker projects in Heroku have it